src/test/skript/junit/ExprChatVisibility.sk

```diff
@@ -0,0 +1,18 @@
+test "ExprChatVisibilityJUnit" when running JUnit:
+	set {_tests::1} to "chat visibility is only commands"
+	set {_tests::2} to "no chat filtering"
+	set {_tests::3} to "can see chat colours"
+	ensure junit test "org.skriptlang.skript.test.tests.syntaxes.expressions.ExprChatVisibilityTest" completes {_tests::*}
+
+on join:
+	set {_test} to "org.skriptlang.skript.test.tests.syntaxes.expressions.ExprChatVisibilityTest"
+	junit test is {_test}
+
+	if player's chat visibility is only commands:
```

Comment:
honestly seeing this syntax makes me think this should be a custom condition. I'd rather see `if player can only see commands in chat` or something to that effect